R/exclude.R

```diff
@@ -79,6 +79,12 @@ parse_exclusions <- function(file, exclude = settings$exclude,
 
   exclusions <- list()
 
+  e <- tryCatch(nchar(lines), error = identity)
+  if (inherits(e, "error")) {
+    # Invalid encoding. Don't parse exclusions.
```

Comment:
maybe a warning here instead?

Reply:
Not sure, `parse_exclusions` is automatically executed on all linted files, so we'd alert in multiple places since there will always be an encoding related lint in that case anyway.

Reply:
true. troubles of reading the PR in alphabetical order 😅